esp/applications/eclservices/jwt_authorization_map.yaml

```diff
@@ -0,0 +1,153 @@
+jwt:
+   root_access:
+      resource: SmcAccess
+      required: Read
+      description: Base access to EclWatch
+   resource_map:
+      WsSMC:
+         Feature:
+         -  path: SmcAccess
+            resource: SmcAccess
+            description: Base access to ESP services
+         -  path: ThorQueueAccess
+            resource: ThorQueueAccess
+            description: Access to thor queues
+         -  path: ConfigAccess"
```

Comment:
Extra quote.

Reply:
Fixed.  Also fixed in esp/applications/eclwatch/jwt_authorization_map.yaml.